presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestingJdbcTypeHandle.java

```diff
@@ -19,20 +19,20 @@
 {
     private TestingJdbcTypeHandle() {}
 
-    public static final JdbcTypeHandle JDBC_BOOLEAN = new JdbcTypeHandle(Types.BOOLEAN, 1, 0);
+    public static final JdbcTypeHandle JDBC_BOOLEAN = new JdbcTypeHandle(Types.BOOLEAN, false, 1, 0);
```

Comment:
please add a test case for unsigned= `true`  as well

Reply:
saw your comment now. 
Just to clarify, were you saying you tried adding a unsigned true test case and H2 doesn't support it?

Reply:
H2 doesn't support unsigned type
So I write another unit test for JdbcTypeHanle, please see my new commit